README.md

```diff
@@ -63,6 +63,7 @@ Major features include:
   for stateful models
 - Provides [Backend API](https://github.com/triton-inference-server/backend) that
   allows adding custom backends and pre/post processing operations
+- Supports writing custom backends in python.
```

Comment:
Assuming we have some high level docs for python-based backends somewhere, let's link it here. Similar to contents of this [comment](https://github.com/triton-inference-server/backend/pull/88#discussion_r1358893542).